### ISSUE TITLE
Running prediction/inference on saved image files

### ISSUE THREAD

**Issue by shubhank008** [XJay18/QuickDraw-pytorch#1]
Neat implementation, I havn't used pytorch a lot so was quite a good learning experience comparing it side by side to the similar keras implementation.  

I had a question though, how do you run inference/prediction on the final model and how do you do that for saved image files (.jpg/.png) as I assume you would need to pre-process the image a bit and normalize it.  

I tried doing it but the prediction was not correct compared to same image used in test/eval data.

Here is my pre-process code
```
x = imread('apple.png', mode='L')

#x = preprocess(x)

x = cv.bitwise_not(x)



x = imresize(x, (28,28))   

x = x.astype('float32')
x /= 255

x = x.reshape(1, input_shape, 1)   # 32,32 or 28,28
```
